source/deployment/advanced-permissions.rst

```diff
@@ -76,21 +76,27 @@ Allows Admins to grant additional permissions to specific users or to a group of
 
 Recipes
 --------
-This section provides some examples of common permissions use cases and how to accomplish them using the Advanced Permissions System Console interface.
+This section provides some examples of common permissions use cases and how to accomplish them using the **Advanced Permissions System Console interface**.
 
 Team Management
 ~~~~~~~~~~~~~~~~
 
-**Restrict adding members to only Admins, in specific teams**
+**Only allow Admins, in specific team, to add members**
 Example: In Team A, only allow Team and System Admins to add new team members. As the default for all other teams, allow all users to add and invite new members.
 
-1. In **System Console** > **Advanced Permissions** in prior versions or **System Console** > **User Management** > **Permissions** in versions after 5.12, edit the **System Scheme**.
-2. In the **All Members** panel, check the box for **Add Team Members**. This sets the system default for all teams.
-3. In **System Console** > **Advanced Permissions** in prior versions or **System Console** > **User Management** > **Permissions** in versions after 5.12, create a new **Team Override Scheme**.
-  i. Name and describe the scheme. For example, ``Authorized Personel Only`` with description ``Restrict adding team members to Team and System Admins.``
+1. Navigate to **System Console** > **User Management** > **Permissions** (or **System Console** > **Advanced Permissions** in versions prior to 5.12)
+2. Select **Edit Scheme**.
+3. In the **All Members > Teams** panel, check the box for **Add Team Members**. This sets the system default for all teams.
+4. Choose **Save**. 
+5. Select the back arrow to return to the **Permission Schemes** menu. 
+
+1. Seect **New Team Override Scheme**.
```

Comment:
Should these be step 6-8?

Reply:
Good question - it's the start of a new series of steps, which is why I numbered it from 1 again. If it makes more sense to start at 6 until 8 I can do that.

Reply:
One of these options works:

1. There should be a title+description in between so it's clear why there are two lists.
2. The second list can be intended similar to how it's [done here](https://github.com/mattermost/docs/blob/8b6080708775f6fe3ed8770ea77e1ed48b36f835/source/deployment/advanced-permissions.rst#team-management).

Reply:
I chose option two, thanks for the suggestions - I didn't think of it like that. :)